src/index.ts

```diff
@@ -34,6 +31,7 @@ export const lambdaTaskRootEnvVar = "LAMBDA_TASK_ROOT";
 export const mergeXrayTracesEnvVar = "DD_MERGE_XRAY_TRACES";
 export const traceExtractorEnvVar = "DD_TRACE_EXTRACTOR";
 export const defaultSiteURL = "datadoghq.com";
+const EXTENSION_PATH = "/opt/extensions/datadog-agent";
```

Comment:
We declare this constant twice, think we can export this function into a utility method so we can use it in one place?
https://github.com/DataDog/datadog-lambda-js/blob/54a6a1436c498009060d6966df8349ea929adc22/src/metrics/extension.ts#L8-L16